aws-codeguruprofiler-profilinggroup/sample-template.json

```diff
@@ -2,10 +2,43 @@
   "AWSTemplateFormatVersion": "2010-09-09",
   "Description": "Sample template for the AWS::CodeGuruProfiler::ProfilingGroup resource.",
   "Resources": {
+    "MyRole": {
+      "Type": "AWS::IAM::Role",
+      "Properties": {
+        "AssumeRolePolicyDocument": {
+          "Version": "2012-10-17",
+          "Statement": [
+            {
+              "Effect": "Allow",
+              "Principal": {
+                "Service": [
+                  "ec2.amazonaws.com"
+                ]
+              },
+              "Action": [
+                "sts:AssumeRole"
+              ]
+            }
+          ]
+        }
+      }
+    },
     "MyProfilingGroup": {
       "Type": "AWS::CodeGuruProfiler::ProfilingGroup",
       "Properties": {
-        "ProfilingGroupName": "MySampleProfilingGroup"
+        "ProfilingGroupName": "MySampleProfilingGroup",
```

Comment:
I think we should have a group without specifying the Permissions and another group with specifying the Permission for testing here.

Reply:
Considering the ```Permissions``` should not be required, I will update the test to contain 2 Profiling Groups as you suggested, thanks.